collectors/python.d.plugin/anomalies/README.md

```diff
@@ -0,0 +1,157 @@
+
+![alt text](https://github.com/andrewm4894/random/blob/master/images/netdata/netdata-anomalies-collector-normal.jpg)
+
+If we then go onto the system and run a command like `stress-ng --all 2` to create some [stress](https://wiki.ubuntu.com/Kernel/Reference/stress-ng), we see some charts begin to have anomaly probabilities that jump outside the typical range. When the anomaly probabilities change enough, we will start seeing anomalies being flagged on the `anomalies.anomaly` chart. The idea is that these charts are the most anomalous right now so could be a good place to start your troubleshooting. 
+
+![alt text](https://github.com/andrewm4894/random/blob/master/images/netdata/netdata-anomalies-collector-abnormal.jpg)
+
+Then, as the issue passes, the anomaly probabilities should settle back down into their 'normal' range again. 
+
+![alt text](https://github.com/andrewm4894/random/blob/master/images/netdata/netdata-anomalies-collector-normal-again.jpg)
+
+## Requirements
+
+- This collector will only work with Python 3 and requires the packages below be installed.
+
+```bash
+# become netdata user
+sudo su -s /bin/bash netdata
+# install required packages for the netdata user
+pip3 install --user requests netdata-pandas numba==0.50.1 scikit-learn pyod
+```
+
+## Configuration
+
+Install the Python requirements above, enable the collector and restart Netdata.
+
+```bash
+cd /etc/netdata/
+sudo ./edit-config python.d.conf
+# Set `anomalies: no` to `anomalies: yes`
+sudo service netdata restart
+```
+
+The configuration for the anomalies collector defines how it will behave on your system and might take some experimentation with over time to set it optimally for your node. Out of the box, the config comes with some [sane defaults](https://www.netdata.cloud/blog/redefining-monitoring-netdata/) to get you started that try to balance the flexibility and power of the ML models with the goal of being as cheap as possible in term of cost on the node resources. 
+
+_**Note**: If you are unsure about any of the below configuration options then it's best to just ignore all this and leave the `anomalies.conf` file alone to begin with. Then you can return to it later if you would like to tune things a bit more once the collector is running for a while and you have a feeling for its performance on your node._
+
+Edit the `python.d/anomalies.conf` configuration file using `edit-config` from the your agent's [config
+directory](https://learn.netdata.cloud/guides/step-by-step/step-04#find-your-netdataconf-file), which is usually at `/etc/netdata`.
+
+```bash
+cd /etc/netdata   # Replace this path with your Netdata config directory, if different
+sudo ./edit-config python.d/anomalies.conf
+```
+
+The default configuration should look something like this. Here you can see each parameter (with sane defaults) and some information about each one and what it does.
+
+```yaml
+# what host to pull data from.
+host: '127.0.0.1:19999'
```

Comment:
Netdata accepts request to `http`, `https` and Unix sockets, should we also specify the protocol here?

Reply:
Added a option to specify `protocol` as 'http' or 'https'